wallet-new/server/Cardano/Wallet/API/V1/Handlers/Transactions.hs

```diff
@@ -24,8 +28,18 @@ handlers = newTransaction
         :<|> allTransactions
         :<|> estimateFees
 
-newTransaction :: Payment -> MonadV1 (WalletResponse Transaction)
-newTransaction _ = single <$> (liftIO $ generate arbitrary)
+newTransaction
+    :: forall ctx m . (V0.MonadWalletTxFull ctx m)
+    => NewPayment -> m (WalletResponse Transaction)
+newTransaction Payment {..} = do
+    let spendingPw = fromMaybe mempty pmtSpendingPassword
+    cAccountId <- migrate (pmtSourceWallet, pmtSourceAccount)
+    addrCoinList <- migrate $ NE.toList pmtDestinations
+  -- Set `OptimiseForSecurityPolicy` as default policy
+    policy <- migrate $ fromMaybe OptimiseForSecurityPolicy pmtGroupingPolicy
```

Comment:
What about creating a `defaultGroupingPolicy :: TransactionGroupingPolicy` function inside `V1.Types` which we can then use here and in all the other places which needs a sensible default? Sounds more refactoring-efficient that scattering the same value in multiple places 😉